multicluster/cmd/service-mirror/main.go

```diff
@@ -164,17 +165,26 @@ func Main(args []string) {
 						}
 					}
 				},
-				UpdateFunc: func(_, obj interface{}) {
-					link, ok := obj.(*v1alpha2.Link)
+				UpdateFunc: func(oldObj, currentObj interface{}) {
+					oldLink, ok := oldObj.(*v1alpha2.Link)
 					if !ok {
-						log.Errorf("object is not a Link: %+v", obj)
+						log.Errorf("object is not a Link: %+v", oldObj)
 						return
 					}
-					if link.GetName() == linkName {
+					currentLink, ok := currentObj.(*v1alpha2.Link)
+					if !ok {
+						log.Errorf("object is not a Link: %+v", currentObj)
+						return
+					}
+					if reflect.DeepEqual(oldLink.Spec, currentLink.Spec) {
+						log.Infof("Link update ignored (only status changed): %s", currentLink.GetName())
+						return
+					}
+					if currentLink.GetName() == linkName {
 						select {
-						case results <- link:
+						case results <- currentLink:
```

Comment:
And on that note, does this cause an INFO message to be emitted elsewhere? It seems like we should note when we are actually processing changes.

Reply:
Yeah we've got below a loop consuming the results channel:
```go
				case link := <-results:
					if link != nil {
						log.Infof("Got updated link %s: %+v", linkName, link)
```